DuckDuckGo/RemoteMessagingClient.swift

```diff
@@ -187,7 +187,6 @@ struct RemoteMessagingClient {
                     case .autoRenewable, .gracePeriod:
                         privacyProIsActive = true
                     case .notAutoRenewable:
-                        privacyProIsActive = true
                         privacyProIsExpiring = true
```

Comment:
We no longer match multiple statuses for expiring users, as we can now match multiple statuses at once using an array in the RMF manifest.